meeteval/wer/wer/cp.py

```diff
@@ -123,15 +124,49 @@ def cp_word_error_rate(
 
     >>> cp_word_error_rate({'r0': 'a b'}, {'h0': 'z', 'h1': 'a e f'})  # Special case for overestimation, that was buggy in the past.
     CPErrorRate(errors=3, length=2, insertions=2, deletions=0, substitutions=1, error_rate=1.5, missed_speaker=0, falarm_speaker=1, scored_speaker=1, assignment=(('r0', 'h1'), (None, 'h0')))
+
+    >>> cp_word_error_rate({'r0': 'a b', 'r1': 'k'}, {'h0': ' ', 'h1': 'a e f'})  # Special case for overestimation, that was buggy in the past.
+    CPErrorRate(errors=3, length=3, insertions=1, deletions=1, substitutions=1, error_rate=1.0, missed_speaker=0, falarm_speaker=0, scored_speaker=2, assignment=(('r0', 'h1'), ('r1', 'h0')))
+
+    >>> from meeteval.io import STM, STMLine
+    >>> r = STM([STMLine.parse('file1 0 r0 0 1 Hello World')])
+    >>> h = STM([STMLine.parse('file1 0 h0 0 1 Hello World')])
+    >>> cp_word_error_rate(r, h)
+    CPErrorRate(errors=0, length=2, insertions=0, deletions=0, substitutions=0, error_rate=0.0, missed_speaker=0, falarm_speaker=0, scored_speaker=1, assignment=(('r0', 'h0'),))
+
+    >>> cp_word_error_rate(['a b c'.split(), 'd e f'.split()], ['a b c'.split(), 'd e f'.split()])
+    CPErrorRate(errors=0, length=6, insertions=0, deletions=0, substitutions=0, error_rate=0.0, missed_speaker=0, falarm_speaker=0, scored_speaker=2, assignment=((0, 0), (1, 1)))
     """
     from meeteval.wer.matching.cy_levenshtein import levenshtein_distance
+    import meeteval.io
+
     def transcription_to_words(x):
```

Comment:
We could define a type for this, similar to what was done in the time-constrained WER

Reply:
Maybe later, when we need that somewhere else.